spec/requests/requests_spec.rb

```diff
@@ -258,7 +258,7 @@
 
       FactoryBot.create(:classification_department_with_tags)
 
-      t = Classification.where(:description => 'Department', :parent_id => 0).includes(:tag).first
+      t = Classification.where(:description => 'Department', :parent_id => nil).includes(:tag).first
```

Comment:
```suggestion
      t = Classification.is_category.includes(:tag).find_by(:description => 'Department')
```

Reply:
right, it was first thought. Second thought was - I am trying to hide possible issue